src/models/programPersonnel.js

```diff
@@ -97,6 +97,14 @@ export default (sequelize, DataTypes) => {
         const { programType } = program;
 
         if (role.toLowerCase() === 'cfo') {
+          if (programType && programType.toLowerCase() === 'ehs') {
+            return 'Chief Financial Officer for Early Head Start';
```

Comment:
Do we want these in a const?

Reply:
Why do you suggest that?

Reply:
Not sure if we will need these in other places ie change in one location.

Reply:
I don't see how these travel. Maybe if we need to add a second spot that would be a good time to make a const